stock_dropshipping_dual_invoice/wizard/stock_invoice_onshipping.py

```diff
@@ -48,9 +48,13 @@ def _need_two_invoices(self):
                 self.env.context['active_id'])
             so = picking.sale_id
             moves = picking.move_lines.filtered('purchase_line_id')[:1]
-            return (so.order_policy == 'picking' and
+            if (so.order_policy == 'picking' and
                     moves.purchase_line_id.order_id.invoice_method ==
-                    'picking')
+                    'picking'):
+                # We still need to investigate if this an MTO or drop-shipping
```

Comment:
Can you please say in the comment that we do that because a MTO incoming picking is associated both to a sale order and to purchase order lines?